.github/workflows/update-contribs.yml

```diff
@@ -0,0 +1,44 @@
+name: Update Contributors
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+jobs:
+    run-meta:
+        runs-on: ubuntu-latest
+        steps:
+        - name: Check out the code
+          uses: actions/checkout@v3
+        - name: Setup Python
+          uses: actions/setup-python@v2
+          with:
+            python-version: '3.10'
+        - name: Upgrade pip
+          run: |
+            # install pip=>20.1 to use "pip cache dir"
+            python3 -m pip install --upgrade pip
+            echo "$PWD"
+            git branch
+        - name: Install pyosmeta and run update contribs
+          env:
+            GITHUB_TOKEN: ${{ github.token }}
+          run: |
+            pip install pyosmeta@git+https://github.com/pyopensci/update-web-metadata
+            echo "$PWD"
+            python3 scripts/parse-contributors.py
+        - name: Commit Files
+          run: |
+            git config --global user.email "leah@pyopensci.org"
+            git config --global user.name "Leah"
+            git status
+            git add _data/contributors.yml
+            git commit -m "Update: Contributor file update"
+        - name: Create Pull Request
+          uses: peter-evans/create-pull-request@v5
+          with:
+            base: main
+            branch: contribs
+            delete-branch: true
+            title: Update New contributors
```

Comment:
```suggestion
            title: Update new contributors
```

Nitpick, but especially since you're going to see a lot of these PRs if you go ahead with this, it seems worth being consistent with capitalization—either title case, or (here) sentence case.

The file is also missing a final EOL (as Git and GitHub are warning you about), but I unfortunately can't make that as a suggestion (AFAIK) due to how GitHub handles leading/trailing whitespace.

Reply:
thanks! i can push up a fix for the EOL issue!